src/events/OmniboxEvents.ts

```diff
@@ -91,11 +90,19 @@ export class OmniboxEvents {
    */
   public static querySuggestSuccess = 'querySuggestSuccess';
   /**
-   * Triggered by the [`Omnibox`]{@link Omnibox} component when a query suggestion is hovered by the mouse or cycled through with the keyboard.
+   * Triggered by the [`Omnibox`]{@link Omnibox} component when a query suggestion has finished rendring.
+   */
+  public static querySuggestRendered = 'querySuggestRendered';
```

Comment:
@fbeaudoincoveo Only those two change from the review of jean-Philippe but feel free to check everything